app/lifecycle/logging.go

```diff
@@ -46,3 +49,29 @@ func InitLogging() {
 
 	slog.Info("ollama app started")
 }
+
+func rotateLogs(logFile string) {
```

Comment:
not critical, some tests would be awesome here

Reply:
Will do.